components/scream/src/control/surface_coupling.hpp

```diff
@@ -56,7 +56,8 @@ class SurfaceCoupling {
                         const int vecComp = -1);
   void register_export (const std::string& fname,
                         const int cpl_idx,
-                        const int vecComp = -1);
+                        const int vecComp = -1,
+                        const bool export_during_init = true);
```

Comment:
Can we add a 2-line comment explaining why we have the `export_during_init` arg? Doesn't have to go into too much detail, just reminding the reader about mct needs.